scss/modules/overlay.scss

```diff
@@ -115,10 +115,25 @@
     }
 }
 
-.overlay-tabs {
+.overlay-subheader {
     background: $white;
     padding: 0 20px;
     border-top: 1px solid $grayLightMedium2;
+    width: 100%;
+    position: relative;
+
+    .language-changer {
+        width: 80px;
+        margin: 10px 0;
+        top: 0;
+        position: absolute;
+        right: 20px;
+
+        &-frame{
+            @extend .language-changer;
```

Comment:
Why does the frame extend the language-changer. Does the sentence "The language-changer frame is a language-changer" make sense? If not the mutal styles should not be set with extend but maybe with:
```
.language-changer,
.language-changer-frame {...}
```